chapters/en/module5.md

```diff
@@ -0,0 +1,297 @@
+Remember that you will open a branch from the stage your project currently is. Are you sure that would have sense to open all of them at the beggining of the project?   
+</opt>
+<opt text='2, 3 and 4 are correct' correct='true'>
+</opt>
+</choice>
+
+
+You clone a repository from Github with a README file to your own computer. After that, you create a new branch locally called <code>app</code> and add to the stage area and commit changes three times. What labels will include your most recent commit (the third one)? 
+
+<choice id = 3>
+<opt text='<code>origin/HEAD</code> and <code>origin/main</code>'>
+<code>origin/HEAD</code> and <code>origin/main</code> are pointing to the latest changes in the remote the moment of pulling.
+</opt>
+<opt text='<code>HEAD -> app</code> ' correct='true'>
+'<code>HEAD</code> is pointing to the last place you make changes and that was in the '<code>app</code> branch
+</opt>
+<opt text='<code>main</code>'>
+In what branch are you working now?
+</opt>
+<opt text=' <code>HEAD -> main</code>'>
+In what branch are you working now? 
+</opt>
+
+</choice>
+
+
+</exercise>
+
+<exercise id='3' title="What is the relationship between the branches?"  type='slides, video'>
+<slides source='module5/module5_02' shot='0' start='3:42' end='4:35'> </slides>
+</exercise>
+
+<exercise id='4' title='Observing the trees'>
+
+**True or False**
+
+The labels <code>origin/main</code> and <code>origin/HEAD</code> always will be associated to the more recent commit in the history of a project that you have just cloned from GitHub.
+
+<choice id = 4>
+<opt text='True'>
+What happens if the last change in the GitHub repository was in a different branch than <code>main</code>? You can try to reproduce this in your computer, creating a new branch in the remote and pulling the information to the local repository!
+</opt>
+<opt text='False'  correct='true' >
+
+The term `origin` refers to the remote repository. If the last change in the remote repository you are cloning was in other branch than <code>main</code> the label in the most recent commit will be (<code>origin/new-branch</code>). <code>origin/HEAD</code> will be always pointing the last commit in the <code>main</code> branch next to <code>origin/main</code>.
+
+</opt>
+</choice>
+
+After running <code> git log --oneline --all --graph </code> in your repository you get as output the following tree:
+
+
+* a949e61 2020-09-13 (HEAD -> master) Update readme
+| * 0a8dfef 2020-09-13 (lecture-todo) List todo items
+|/
+* 6e69947 2020-09-13 (origin/master, origin/HEAD) Add license
+* 0d5a6f7 2020-09-13 Initial commit
+
+
+Select the options that are TRUE
+
+<choice id = 5>
+<opt text='The most recent commit on GitHub was performed on the <code>lecture-todo</code> branch'>
+You can recognize the latest commit on GitHub with the label <code> origin/HEAD </code>.
+</opt>
+<opt text='There is a branch called <code> lecture-todo </code> that has not been pushed to GitHub yet ' correct='true'>
```

Comment:
```suggestion
<opt text='There are changes to the <code> lecture-todo </code> branch that has not been pushed to GitHub yet ' correct='true'>
```

Technically the branch could exist but be older than the three commits we see here